### PR TITLE
Switching Pointfinder Selection to Drop Down List

### LINKED ISSUE #19: Move selection of PointFinder organism to a required parameter in IRIDA
Right now, the parameter used to enable the use of the PointFinder database and select an organism is hidden among all the other parameters that can be adjusted and so is not as obvious that it needs to be configured:

image

I believe we should move this parameter to the front of the parameters list that gets displayed to users in IRIDA so that it's more obvious it needs to be set, similar to selecting an RGI database:

image

The way this can be accomplished is to update the `use_pointfinder` parameter setting:

https://github.com/phac-nml/irida-plugin-amr-detection/blob/52e3ec2f1e086d48e7d19325a503df4fc319a42d/src/main/resources/workflows/0.3.0/irida_workflow.xml#L42-L44

such that you add `required="true"` and then add multiple sub `<choice>salmonella</choice>` entries (one for every validated organism in staramr: **campylobacter, enterococcus_faecalis, enterococcus_faecium, escherichia_coli, helicobacter_pylori, salmonella**, and **disabled**).

For example:

```xml
<parameter name="staramr-2-use_pointfinder" defaultValue="disabled" required="true">
  <choices>
    <choice>disabled</choice>
    <choice>salmonella</choice>
    <choice>campylobacter</choice>
  </choices>
  <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.9.1+galaxy0" parameterName="use_pointfinder"/>
</parameter>
```

This is how the `biohansel` pipeline in IRIDA operates:

image

Code for BioHansel: 
https://github.com/phac-nml/irida/blob/edf0556af75021c309b1f3fe5328a1a28a1e1137/src/main/resources/ca/corefacility/bioinformatics/irida/model/workflow/analysis/type/workflows/BioHansel/2.0.0/irida_workflow.xml#L12-L18

### PR DESCRIPTION
**Pull request by emarinier**
As per #19 

image

### CODE REVIEW

File: src/main/resources/workflows/0.3.0/irida_workflow.xml [hunk 1]
```diff
@@ -39,7 +39,16 @@
...
+      <choices>
+	<choice name="disabled" value="disabled"/>
+	<choice name="campylobacter" value="campylobacter"/>
+        <choice name="enterococcus_faecalis" value="enterococcus_faecalis"/>
+        <choice name="enterococcus_faecium" value="enterococcus_faecium"/>
+        <choice name="escherichia_coli" value="escherichia_coli"/>
+        <choice name="helicobacter_pylori" value="helicobacter_pylori"/>
+        <choice name="salmonella" value="salmonella"/>
+      </choices>
       <toolParameter toolId="toolshed.g2.bx.psu.edu/repos/nml/staramr/staramr_search/0.9.1+galaxy0" parameterName="use_pointfinder"/>
```
Comment (apetkau) [inline src/main/resources/workflows/0.3.0/irida_workflow.xml]: I think the `parameterName` here needs to be set to `pointfinder_db.use_pointfinder` in order to match the conditional name in the Galaxy tool

https://github.com/phac-nml/galaxy_tools/blob/c32ec4a9284ec939d001bd7d7d551c15feaa5333/tools/staramr/staramr_search.xml#L83

**Review by apetkau — CHANGES_REQUESTED** [review]
Thanks so much Eric. This looks great. Just one comment.

File: src/main/resources/workflows/0.3.0/irida_workflow.xml (hunk 1 above)
Comment (emarinier) [reply src/main/resources/workflows/0.3.0/irida_workflow.xml]: Good catch! Fixed.

**Review by apetkau — APPROVED** [review]
Thanks so much Eric. This looks great. Works perfectly.

**Pull request merged**